go-controller/pkg/config/config.go

```diff
@@ -1415,6 +1470,69 @@ func initConfigWithPath(ctx *cli.Context, exec kexec.Interface, saPath string, d
 	return retConfigFile, nil
 }
 
+// validateFlowCollectors validates the flow collectors of Monitoring object
+// are strings containing <ip:port> separated by commas, e.g. "192.168.1.1:80,192.168.1.2:81"
+func validateFlowCollectors(monitoring MonitoringConfig) error {
+	if monitoring.NetFlowTargets != "" {
+		collectors := strings.Split(monitoring.NetFlowTargets, ",")
+		for _, v := range collectors {
+			splittedCollector := strings.Split(v, ":")
```

Comment:
@rcarrillocruz `net.SplitHostPort()` takes care of handling IPv6 too by requiring the address to be bracketed so it doesn't conflict with the : port separator.

Reply:
Neat!

Reply:
Done